front-end/src/components/AnalyzersAndSeries.js

```diff
@@ -24,12 +23,20 @@ const AnalyzersAndSeries = ({ dimensions, columns, setDisplayPage, setShowModal,
 
     var seriesInfo = []
 
-    const addSeries = () => {
+    const addSeries = (isSeries) => {
+        
 
         var selectColumns = [];
         for (let i = 0; i < dimensions; i++) {
             selectColumns.push(JSON.parse(document.getElementsByClassName(i)[0].value));
         }
+        if (isSeries) {
+            setSuccessMessage("Series Added");
+            success();
+        } else {
+            setSuccessMessage("Graph Created");
+            success();
+        }
```

Comment:
success is called either way, can just be called once after the if else

Reply:
Yup u right